src/Store/KeyBindingsStoreConnector.re

```diff
@@ -169,6 +169,11 @@ let start = () => {
         command: "redo",
         condition: "editorTextFocus" |> WhenExpr.parse,
       },
+      {
+        key: "<C-S>",
+        command: "workbench.action.files.save",
+        condition: "insertMode" |> WhenExpr.parse,
```

Comment:
Do we not want this in any mode? i.e. `editorTextFocus` instead of `insertMode`.